Add hw9

MastenaM opened the pull request.

MastenaM committed:
Add hw9

pacifikus reviewed (changes requested):
Добрый день!

Спасибо за работу! В целом все хорошо, но, если честно, пока не очень понятно, как вы запускаете свое приложение - стандартно через cli и .py файл или напрямую через ноутбук - пока видно только ошибку в запуске команды...
Уточните, пожалуйста, получилось ли у вас запустить приложение, и если да, то каким способом? Можно приложить скриншот, если так будет проще

pacifikus commented:
Также отмечу, что логи от установки библиотек лучше удалять из ноутбука перед коммитом, так как они затрудняют восприятие, не давая полезной информации.

MastenaM commented:
Здравствуйте. 
У меня так и не получилось запустить.

pacifikus commented:
А вы работаете в колабе или локально запускаете jupyter ноутбук?

MastenaM commented:
В коллабе. У меня есть jupiter notebook, но он не очень удобен.

pacifikus commented:
В колабе напрямую будет не так просто реализовать запуск как в этом ДЗ, так и в следующем. Можно попробовать посмотреть способы по ссылкам ниже:
https://discuss.streamlit.io/t/how-to-launch-streamlit-app-from-google-colab-notebook/42399
https://stackoverflow.com/questions/63833593/how-to-run-fastapi-uvicorn-in-google-colab

Но если вдруг не получится, то самый простой путь - сделать локальное развертывание через cli и .py скрипт,к ак это рассматривалось на занятии...

MastenaM commented:
Спасибо. Пойду пробовать)

MastenaM committed:
Переделала задание 9 

Переделала задание 9 так, чтобы запустилось через коллаб, как указано в статье, но все равно не получается запустить

pacifikus reviewed (changes requested):
Добрый день!

Спасибо за работу!

Пока что вы не очень точно следовали руководству - в нем контент streamlit-приложения  записывается в скрипт `app.py`, но вы этого не делаете, а потом все равно запускаете streamlit с этим скриптом... Также упомяну, что для успешного запуска в app.py должны быть все импорты, код чтения модели из файла, код предсказания с помощью загруженной модели и контент приложения. Прямо как в скриптах из лекций.

Также упомяну, что для того, чтобы посмотреть на ваше приложение, нужно перейти по ссылке после запуска туннеля и вставить полученный ip как tunnel password

![image](https://github.com/user-attachments/assets/b28b7a04-60ad-46c9-9e44-c1769b2d0eba)
